src/hyrdate.js

```diff
@@ -1,20 +1,30 @@
 class Hydration {
-  constructor(hydrationData) {
-    this.userId = hydrationData.userID,
-    this.date = hydrationData.date,
-    this.numOunces = hydrationData.numOunces
+  constructor(hydrationData, userID) {
+    this.hydrationData = hydrationData;
+    this.userID = userID;
   }
 
-  calculateAverageDailyConsumption() {
+  calculateTotalConsumption(id) {
```

Comment:
For this one the instructions say that it should be able to calculate the overalll average daily consumption for the user not the total.  So it will have to be modified a bit.

Reply:
thank you for catching that! I thought I had read it wrong the first time!